### PR TITLE
KDE publisher page

### LINKED ISSUE #1817: KDE publisher page

### PR DESCRIPTION
**Pull request by therealjuan**
Fixes #1817 

### QA Steps
1. Pull the branch or run the demo service
2. Visit `/publisher/kde`
3. #winning

### CODE REVIEW

**Comment by bartaz** [comment]
@evandandrea @therealjuan Is 'publisher since' date correct? something we verified? It is the same as in jetbrains, so it may be just a copy/paste...

Screenshot 2019-04-17 at 14 49 57

File: webapp/store/views.py [hunk 1]
```diff
@@ -241,20 +241,22 @@ def _get_file(file):
                 os.path.join(flask.current_app.root_path, file), "r"
             ) as stream:
                 data = yaml.load(stream)
-        except Exception:
+        except Exception as e:
             data = None
+            print(e)
```
Comment (bartaz) [inline webapp/store/views.py]: Is it leftover from debugging?

Comment (therealjuan) [reply webapp/store/views.py]: yes!

**Comment by therealjuan** [comment]
@bartaz spot on, just updated to January 2017 as I found some info on the internet about this :)

**Pull request merged**